alpaca/rest.go

```diff
@@ -742,6 +744,121 @@ func (c *Client) GetAsset(symbol string) (*Asset, error) {
 	return &asset, nil
 }
 
+type GetOptionContractsRequest struct {
+	UnderlyingSymbols string           `json:"underlying_symbols"`
```

Comment:
Btw this is also optional, but see my other comment.

Reply:
Oops, the changes from above will fix this.